fn_cve_search/README.md

```diff
@@ -0,0 +1,139 @@
+#About This Package
+
+This Common Vulnerability Exposures function provides a way to search for vulnerabilities from CVE Database, 
+     function offers below mentioned ways to search for vulnerabilities.
+
+* Browse
+* Search
+* Specific CVE ID
+* Last 30 CVE's
+* CVE Database Information
+ 
+For More Information see https://www.circl.lu/services/cve-search/
+
+#Prerequisites:
+
+* resilient version 31 or later
+* resilient_circuits version 30 or later
+    
+#Resilient Installation
+
+This package requires that it is installed on a RHEL or CentOS platform and uses the resilient-circuits framework.
+Install this package with 'pip', such as:
+
+`pip install fn_cve_search-<version>.tar.gz`
+
+To import the function and example rule and workflows into Resilient, run the following command:
+
+`resilient-circuits customize`
+
+Answer the prompts for the package to import.
+
+To import the data table into Resilient after customization, follow below instructions:
```

Comment:
The layout can be extended to add the CVE function datatable by following these instructions:

Reply:
ok